packages/manager/apps/container/package.json

```diff
@@ -29,13 +29,14 @@
     "@ovh-ux/shell": "^0.0.0 || ^1.0.0",
     "@ovh-ux/ufrontend": "^1.1.1 || ^2.0.0",
     "@ovh-ux/ui-kit": "^5.1.0",
-    "bootstrap": "^4.4.1",
+    "bootstrap": "4.6.0",
     "date-fns": "^2.24.0",
     "i18next": "^20.4.0",
     "i18next-http-backend": "^1.3.0",
     "lodash-es": "^4.17.15",
     "prop-types": "^15.7.2",
     "react": "^17.0.2",
+    "react-bootstrap": "^2.1.2",
```

Comment:
`react-bootstrap` v2.x seems to be for `bootstrap` v5.x but you are using `bootstrap` v4.6 (https://github.com/react-bootstrap/react-bootstrap#bootstrap-compatibility)